src/connectors/acp_core/base_connector.py

```
@@ -1842,28 +1882,20 @@ async def _acquire_runtime_request_lock(self, runtime: RuntimeT) -> None:
         if process.poll() is not None:
             return True
         try:
-            await asyncio.wait_for(
-                asyncio.to_thread(process.wait),
-                timeout=timeout_s,
-            )
+            await asyncio.wait_for(asyncio.to_thread(process.wait), timeout=timeout_s)
             return True
         except asyncio.TimeoutError:
             return False
         except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)